index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Loan System</title>
+    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="top-bar">
+    <ul class="top-bar-items">
+      <li class="top-bar-icon"><i class="fab fa-facebook-f"></i></li>
+      <li class="top-bar-icon"><i class="fab fa-twitter"></i></li>
+      <li class="top-bar-icon"><a href="#">english</a></li>
+      <li class="top-bar-icon"><a href="#">my page</a></li>
+      <li class="top-bar-icon">logout</li>
+    </ul>
+  </div>
+    <nav>
+        <div class="logo"><a href="index.html">My Logo</a></div>
```

Comment:
[OPTIONAL]
- I recommend using an image as a logo instead of text as it looks weird in some of the place.